tsfc/coffee.py

```diff
@@ -195,7 +190,7 @@ def statement_evaluate(leaf, parameters):
         table[abs(table + 1.0) < epsilon] = -1.0
         table[abs(table - 0.5) < epsilon] = 0.5
         table[abs(table + 0.5) < epsilon] = -0.5
-        init = coffee.SparseArrayInit(table, parameters.precision, nz_bounds)
+        init = coffee.ArrayInit(table, parameters.precision)
```

Comment:
We assume this basically doesn't matter because the big zero blocks are gone in this implementation anyway.